protocol/dubbo/pool.go

```diff
@@ -53,15 +55,27 @@ var (
 )
 
 func newGettyRPCClientConn(pool *gettyRPCClientPool, protocol, addr string) (*gettyRPCClient, error) {
-	c := &gettyRPCClient{
-		protocol: protocol,
-		addr:     addr,
-		pool:     pool,
-		gettyClient: getty.NewTCPClient(
+	var gettyClient getty.Client
+	if pool.sslEnabled {
```

Comment:
```suggestion
	var (
	      gettyClient getty.Client
	      sslEnabled bool
        )
	sslEnabled = pool.sslEnabled
	if sslEnabled {
```

make a comment variable.